bin/arthurd

```diff
@@ -199,22 +198,26 @@ def create_common_arguments_parser(defaults):
                         help=argparse.SUPPRESS)
     parser.add_argument('--es-index', dest='es_index',
                         help=argparse.SUPPRESS)
-    parser.add_argument('--log-path', dest='log_path',
-                        default=os.path.expanduser('~/.arthur/logs/'),
-                        help=argparse.SUPPRESS)
     parser.add_argument('--archive-path', dest='archive_path',
                         default=os.path.expanduser(ARCHIVES_DEFAULT_PATH),
                         help=argparse.SUPPRESS)
     parser.add_argument('--no-archive', dest='no_archive',
                         action='store_true',
                         help=argparse.SUPPRESS)
-    parser.add_argument('--no-daemon', dest='no_daemon',
-                        action='store_true',
-                        help=argparse.SUPPRESS)
     parser.add_argument('--pubsub-channel', dest='pubsub_channel',
                         default=CH_PUBSUB,
                         help=argparse.SUPPRESS)
 
+    group_log = parser.add_argument_group('Log handling')
+    exclusive_log = group_log.add_mutually_exclusive_group()
+
+    exclusive_log.add_argument('--log-path', dest='log_path',
+                               default=os.path.expanduser('~/.arthur/logs/'),
+                               help=argparse.SUPPRESS)
+    exclusive_log.add_argument('--no-daemon', dest='no_daemon',
+                               action='store_true',
+                               help=argparse.SUPPRESS)
+
```

Comment:
Taking into account @sanacl comment, the behaviour is right but this part is misleading. The option `--no-daemon` is not related to the logs, so it shouldn't be under that group. Specially because if you run `arthurd -h` that option will be under `Log handling` section.

I think we need a better solution for this, probably just ignoring `--log-path` when `--no-daemon` is set. You can include a comment like `no effect when running in the foreground` in the help section.